tests/components/watergate/test_coordinator.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for the Watergate coordinator."""
+
+from homeassistant.components import valve
+from homeassistant.const import ATTR_ENTITY_ID, SERVICE_CLOSE_VALVE, SERVICE_OPEN_VALVE
+from homeassistant.core import HomeAssistant
+
+
+async def test_change_valve_state(
```

Comment:
move this to test_valve.py and I'd suggest you throw a snapshot test in there

Reply:
I've added one snapshot assertion. Not for all states. To be honest, in the entity, we do not have anything more than the state, so one is enough for me, the rest of the state changes I am testing with other assertions.